server/routes/detail.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const router = express.Router();
+const Detail = require('../../db/Detail.js');
+
+//get detail info for all houses
+router.get('/homes/all', (req, res) => {
```

Comment:
```suggestion
router.get('/homes', (req, res) => {
```
I think the all is redundant here.

Reply:
I agree. Fixed!